### DISCUSSION TITLE
Issue in click option

### DISCUSSION THREAD

**Discussion by Advinsteve** [laravel/framework#46485] [General]
<?php

namespace Tests\Browser;

use Tests\DuskTestCase;
use Laravel\Dusk\Browser;
use Laravel\Dusk\Browser\driver;
use Facebook\WebDriver\WebDriverBy;
use Illuminate\Foundation\Testing\DatabaseMigrations;

class TodoTest extends DuskTestCase
{
    /**
     * A Dusk test example.
     *
     * @return void
     */
    public function testExample()
    {
        
        $this->browse(function (Browser $browser) {

          
           
           $a=$browser->driver->findElements(WebDriverBy::id('com.pcloudy.appiumdemo:id/flightButton'));
           $b=strval($a);
           echo $b;
           $browser->click($b);
            
        
        //     sleep(10);
        });

    }
}

I like to click the button which I locate from webdriver by id

**Comment by Angel5a** [comment]
```
$a->click();
```
Should work.
